pkg/controller/useraccount/useraccount_controller.go

```diff
@@ -64,18 +82,136 @@ func (r *ReconcileUserAccount) Reconcile(request reconcile.Request) (reconcile.R
 	reqLogger.Info("Reconciling UserAccount")
 
 	// Fetch the UserAccount instance
-	instance := &toolchainv1alpha1.UserAccount{}
-	err := r.client.Get(context.TODO(), request.NamespacedName, instance)
+	userAcc := &toolchainv1alpha1.UserAccount{}
+	err := r.client.Get(context.TODO(), types.NamespacedName{Namespace: config.GetOperatorNamespace(), Name: request.Name}, userAcc)
 	if err != nil {
 		if errors.IsNotFound(err) {
 			// Request object not found, could have been deleted after reconcile request.
 			// Owned objects are automatically garbage collected. For additional cleanup logic use finalizers.
 			// Return and don't requeue
 			return reconcile.Result{}, nil
 		}
-		// Error reading the object - requeue the request.
+		return reconcile.Result{}, err
+	}
+
+	var created bool
+	var user *userv1.User
+	if user, created, err = r.ensureUser(userAcc); err != nil || created == true {
+		return reconcile.Result{}, err
+	}
+
+	var identity *userv1.Identity
+	if identity, created, err = r.ensureIdentity(userAcc); err != nil || created == true {
+		return reconcile.Result{}, err
+	}
+
+	if created, err = r.ensureMapping(userAcc, user, identity); err != nil || created == true {
 		return reconcile.Result{}, err
 	}
 
```

Comment:
And update the status to `provisioned` here.

Reply:
I guess, we can do that once the namespace provisioned for this user as part of https://jira.coreos.com/browse/CRT-74 .. fine ??

Reply:
Well, it's still a completed provisioning. We just don't do the namespace provisioning yet in this PR. And it's a few lines of code.

Reply:
done [d79d00f](https://github.com/codeready-toolchain/member-operator/pull/20/commits/d79d00f6cf3d7eed5890446ea9c32e75f7563fc6)